operator/src/main/resources/scripts/utils_base.sh

```diff
@@ -0,0 +1,256 @@
+  trace "id = '`id`'"
+  local keyword="$1"
+  shift
+  local indir
+  local val_indir
+  for indir in $*; do
+    eval "val_indir=\"\$${indir}\""
+    [ -z "${val_indir}" ] && continue
+    trace "Directory trace for $indir=${val_indir} ($keyword)"
+    local cnt=0
+    local odir=""
+    local cdir="${val_indir}/*"
+    while [ ${cnt} -lt 30 ] && [ ! "$cdir" = "$odir" ]; do
+      echo "  ls -ld $cdir:"
+      ls -ld $cdir 2>&1 | sed 's/^/    /'
+      odir="$cdir"
+      cdir="`dirname "$cdir"`"
+      cnt=$((cnt + 1))
+    done
+  done
+}
+
+# timestamp
+#   purpose:  echo timestamp in the form yyyymmddThh:mm:ss.mmm ZZZ
+#   example:  20181001T14:00:00.001 UTC
+function timestamp() {
+  local timestamp="`date --utc '+%Y-%m-%dT%H:%M:%S %N %s %Z' 2>&1`"
+  if [ ! "${timestamp/illegal/xyz}" = "${timestamp}" ]; then
+    # old shell versions don't support %N or --utc
+    timestamp="`date -u '+%Y-%m-%dT%H:%M:%S 000000 %s %Z' 2>&1`"
+  fi
+  local ymdhms="`echo $timestamp | awk '{ print $1 }'`"
+  # convert nano to milli
+  local milli="`echo $timestamp | awk '{ print $2 }' | sed 's/\(^...\).*/\1/'`"
+  local secs_since_epoch="`echo $timestamp | awk '{ print $3 }'`"
+  local millis_since_opoch="${secs_since_epoch}${milli}"
+  local timezone="`echo $timestamp | awk '{ print $4 }'`"
+  echo "${ymdhms}.${milli} ${timezone}"
+}
+
+# 
+# checkEnv [-q] envvar1 envvar2 ...
+#
+#   purpose: Check and trace the values of the provided env vars.
+#            If any env vars don't exist or are empty, return non-zero
+#            and trace an '[SEVERE]'.
+#            (Pass '-q' to suppress FINE tracing.)
+#
+#   sample:  checkEnv HOST NOTSET1 USER NOTSET2
+#            @[2018-10-05T22:48:04.368 UTC][FINE] HOST='esscupcakes'
+#            @[2018-10-05T22:48:04.393 UTC][FINE] USER='friendly'
+#            @[2018-10-05T22:48:04.415 UTC][SEVERE] The following env vars are missing or empty:  NOTSET1 NOTSET2
+#
+function checkEnv() {
+  local do_fine="true"
+  if [ "$1" = "-q" ]; then 
+    do_fine="false"
+    shift
+  fi
+  
+  local not_found=""
+  local val_1
+  while [ ! -z "${1}" ]; do 
+    eval "val_1=\"\$${1}\""
+    if [ -z "${val_1}" ]; then
+      not_found="$not_found ${1}"
+    else
+      [ "$do_fine" = "true" ] && trace FINE "${1}='${val_1}'"
+    fi
+    shift
+  done
+  if [ ! -z "${not_found}" ]; then
+    trace SEVERE "The following env vars are missing or empty: ${not_found}"
+    return 1
+  fi
+  return 0
+}
+
+#
+# initCommonMount
+#   purpose: Execute the COMMON_MOUNT_COMMAND specified as part of the common mount init container.
+#            If the specified COMMON_MOUNT_COMMAND is empty, it logs an error message and returns.
+#            If the COMMON_MOUNT_PATH directory doesn't exist or is empty, it logs error and returns.
+#            If the command execution fails, it logs errror message with failure details. Otherwise it
+#            logs a success message with details.
+#            See also 'commonMount.sh'.
```

Comment:
Minor: Add something like: See also checkCommonMount in 'utils.sh.'
Minor: Similarly update the checkCommonMount comment to point  to initCommonMount in 'base_utils.sh'...